webapp/src/Entity/Contest.php

```diff
@@ -170,6 +170,52 @@ class Contest extends BaseApiEntity
      */
     private $b = 0;
 
+    /**
+     * @var boolean|null
+     * @ORM\Column(type="boolean", name="process_awards",
+     *     options={"comment"="Are there awards for this contest?","default"=0},
+     *     nullable=false)
+     * @Serializer\Exclude()
+     */
+    private $processAwards = false;
+
+    /**
+     * @ORM\ManyToMany(targetEntity="App\Entity\TeamCategory", inversedBy="contests")
+     * @ORM\JoinTable(name="contestteamcategoryforawards",
+     *                joinColumns={@ORM\JoinColumn(name="cid", referencedColumnName="cid", onDelete="CASCADE")},
+     *                inverseJoinColumns={@ORM\JoinColumn(name="categoryid", referencedColumnName="categoryid", onDelete="CASCADE")}
+     *               )
+     * @Serializer\Exclude()
+     */
+    private $awardsCategories;
+
+    /**
+     * @var int|null
+     * @ORM\Column(type="smallint", length=3, name="gold_awards",
+     *     options={"comment"="Number of gold medals","unsigned"="true","default"=0},
```

Comment:
Maybe we should call it also `awards` here and not `medals`?

Reply:
Sure.